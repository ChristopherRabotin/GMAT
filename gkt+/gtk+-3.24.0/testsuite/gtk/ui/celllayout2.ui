<!-- test invalid cell layout attribute value -->
<interface>
  <object class="GtkIconView">
    <child>
      <object class="GtkCellRendererText">
      </object>
      <attributes>
        <attribute name="text">blue</attribute>
      </attributes>
    </child>
  </object>
</interface>
