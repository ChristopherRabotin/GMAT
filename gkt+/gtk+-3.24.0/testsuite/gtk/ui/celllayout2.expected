ERROR: gtk-builder-error-quark 6
.:9:1 Could not parse integer 'blue'
